src/components/search/Search.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+
+const architectsList = [
+  {
+    id: 1,
+    name: 'John',
+    surname: 'Doe',
+    city: 'Minsk',
+  },
+  {
+    id: 2,
+    name: 'Mike',
+    surname: 'Hike',
+    city: 'Hrodno',
+  },
+  {
+    id: 3,
+    name: 'Jane',
+    surname: 'Doe',
+    city: 'Vitebsk',
+  },
+  {
+    id: 4,
+    name: 'Sarah',
+    surname: 'Smith',
+    city: 'Homel',
+  },
+
+];
+
+class Search extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      architects: architectsList,
+      query: '',
+    };
+  }
+
+  searchHandler = (event) => {
+    this.setState({ query: event.target.value });
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  searchFor(query) {
```

Comment:
this method should be static, it will fix the error

Reply:
fix: commit da43aa90d6342c900747525afd457b856b16e49a